script.js

```diff
@@ -62,14 +80,48 @@ function addKeys() {
 
 function addNotes() {
 	let notes = document.createElement("p");
-	notes.innerHTML = "Windows OS";
+	notes.innerHTML = "Windows OS<br>Language change: Shift + Alt";
 	wrapper.appendChild(notes);
 }
 
 document.addEventListener("keydown", function (event) {
-	document.getElementById(event.code).classList.add("key-active");
+
+	if (document.getElementById(event.code)) {
+		document.getElementById(event.code).classList.add("key-active");

```

Comment:
Please, don't use magic strings here and below. Create some kind of dictionary for keys and classes.